crates/fluvio-connector-package/Cargo.toml

```diff
@@ -11,11 +11,19 @@ description = "Fluvio Connector Package"
 default = ["toml"]
 
 [dependencies]
-bytesize = "1.1.0"
-serde = { version = "1.0", default-features = false, features = ["derive"] }
-serde_yaml = "0.8.18"
+anyhow = { workspace = true }
+bytesize = { workspace = true}
 humantime-serde = "1.1.1"
+lazy_static = "1.4.0"
+openapiv3 = { git = "https://github.com/galibey/openapiv3", rev = "bdd22f046d2bc19ede257504645d31f835545222", default-features = false }
+once_cell = { workspace = true }
+regex = { workspace = true }
+serde = { workspace = true, default-features = false, features = ["derive"] }
+serde_yaml = "0.8.18"
```

Comment:
this should be updated to use workspace dependency (0.9.0) not doing in this PR because I think that it would require some code changes